CMakeLists.txt

```diff
@@ -834,8 +834,10 @@ message(STATUS "Run tests with ICD: ${TESTS_USE_ICD}")
 
 ######################################################################################
 
-if(DEFINED INSTALL_OPENCL_HEADERS)
+if(INSTALL_OPENCL_HEADERS)
   message(STATUS "Install POCL's OpenCL headers: ${INSTALL_OPENCL_HEADERS}")
+elseif(DEFINED INSTALL_OPENCL_HEADERS AND NOT INSTALL_OPENCL_HEADERS)
+  message(STATUS "Not install POCL's OpenCL headers: ${INSTALL_OPENCL_HEADERS}")
```

Comment:
Can you change the message simply to "Not installing OpenCL headers."

Reply:
Done.